### DISCUSSION TITLE
posts/tls-cryptography-1-overview/

### DISCUSSION THREAD

**Discussion by giscus** [humminglab/blog.humminglab.io#7] [Announcements]
# posts/tls-cryptography-1-overview/

HTTPS의 SSL/TLS를 이해하는 것은 생각만큼 쉽지 않다. React, kubernetes 같이 오픈 생태계에서 핫 한 분야들은 다양한 시각으로 설명한 자료들이 많아 여러 방법으로 이해할 수도 있겠지만, 수십여년간 수학자, 암호학자들에게서 다듬어진 암호학에 대해서는 말랑말랑한 자료를 찾기가 쉽지는 않다. 좋은 자료라고 찾아 보아도 읽다 보면 이내 수많은 용어들과 수학 이론에 막혀 버리곤 한다. 그렇다고 제대로된 이해없이 SSL/TLS 나 암호화 알고리즘을 사용하게 되면 작은 실수로 인하여 보안에 심각한 문제를 만들 수도 있다.
여기에서는 일반 개발자를 위한 관점으로 암호 알고리즘과 TLS/SSL을 정리해 보기로 한다.

https://blog.humminglab.io/posts/tls-cryptography-1-overview/

**Comment by liebespaar93** [comment]
정리해주셔서 감사합니다~
아직 제대로 이해를 못하긴 했지만 노력중!

**Reply by aqwerf** [reply]
저도 공부할 겸 정리 하였는데, 도움이 되시면 좋겠어요.
